server/provider/currenciesProvider.go

```diff
@@ -0,0 +1,61 @@
+package provider
+
+import "github.com/multiversx/mx-chain-rosetta/server/resources"
+
+type currenciesProvider struct {
```

Comment:
Question for review: or, perhaps `currenciesHolder`?

Reply:
I think both are fine